main.nf

```diff
@@ -1360,8 +1419,73 @@ process circularmapper{
     
 }
 
+process bowtie2 {
+    label 'mc_medium'
+    tag "${libraryid}"
+    publishDir "${params.outdir}/mapping/bt2", mode: 'copy'
+
+    when: params.mapper == 'bowtie2'
```

Comment:
when: ideally after output and before script

Reply:
I blame you for that one. I copied directly from the BWA process ;). I've updated all mappers accordingly for you ;).